rds_database_cluster/rds_database_cluster.tf

```diff
@@ -39,6 +39,7 @@ resource "aws_rds_cluster" "rds_db_cluster" {
   port                                = "${lookup(var.optional_parameters, "port", 3306)}"
   vpc_security_group_ids              = ["${var.vpc_security_group_ids}"]
   snapshot_identifier                 = "${lookup(var.optional_parameters, "snapshot_identifier", "")}"
+  final_snapshot_identifier           = "${lookup(var.optional_parameters, "final_snapshot_identifier", "")}"
```

Comment:
its almost worth making this required, or at least require an explicit opt-out, if that's possible

Reply:
You're right, I will changed that as required.